src/api/flaskr/service/shifu/block_to_mdflow_adapter.py

```
@@ -0,0 +1,193 @@
+
+
+def __convert_goto_to_mdflow(goto: GotoDTO, variable_map: dict[str, str]) -> str:
+    if hasattr(goto, "label") and goto.label:
+        return __convert_button_with_value_to_mdflow(goto.label, "_sys_goto")
+    else:
+        # Create a default label for goto
+        label_dto = LabelDTO(lang={"zh-CN": "跳转"})
+        return __convert_button_with_value_to_mdflow(label_dto, "_sys_goto")
+
+
+def __convert_phone_to_mdflow(phone: PhoneDTO, variable_map: dict[str, str]) -> str:
```

Ruff (0.13.1)

Unused function argument: `variable_map`

(ARG001)